solutions/standard/DA-types.md

```diff
@@ -0,0 +1,145 @@
+
+**Disk Auto-Scaling Configuration**
+The disk object within `auto_scaling` contains the following properties:
+
+- `capacity_enabled (optional, default=false):` Indicates whether disk capacity auto-scaling is enabled.
+
+- `free_space_less_than_percent (optional, default=10):` Specifies the threshold percentage of free disk space below which auto-scaling is triggered.
+
+- `io_above_percent (optional, default=90):` Sets the IO(Input/Output) usage percentage above which auto-scaling is triggered.
+
+- `io_enabled (optional, default=false):` Indicates whether IO-based auto-scaling is enabled.
+
+- `io_over_period (optional, default="15m"(15 minutes)):` Defines the period over which IO usage is evaluated for auto-scaling.
+
+- `rate_increase_percent (optional, default=10):` Specifies the percentage increase in disk capacity when auto-scaling is triggered.
+
+- `rate_limit_mb_per_member (optional, default=3670016 MB):` Sets the limit on the rate of disk increase per member (in megabytes).
+
+- `rate_period_seconds (optional, default=900 seconds (15 minutes)):` Defines the period (in seconds) over which the rate limit is applied.
+
+- `rate_units (optional, default="mb" (megabytes)):` Specifies the units used for rate increase.
+
+<br>**Memory Auto-Scaling Configuration**: The memory object within auto_scaling contains the following properties:
+
+- `io_above_percent (optional, default= 90):` Sets the IO usage percentage above which memory auto-scaling is triggered.
+
+- `io_enabled (optional, default= false):` Indicates whether IO-based auto-scaling for memory is enabled.
+
+- `io_over_period (optional, default= "15m" (15 minutes)):` Defines the period over which IO usage is evaluated for memory auto-scaling.
+
+- `rate_increase_percent (optional, default= 10):` Specifies the percentage increase in memory capacity when auto-scaling is triggered.
+
+- `rate_limit_mb_per_member (optional, default= 114688 MB):` Sets the limit on the rate of memory increase per member (in megabytes).
+
+- `rate_period_seconds (optional, default= 900 seconds (15 minutes)):` Defines the period (in seconds) over which the rate limit is applied for memory.
+
+- `rate_units (optional, default= "mb" (megabytes)):` Specifies the units used for rate increase.
+
+**Sample Example:** Below example is to illustrate how to use both disk and memory configurations.
+
+```json
+{
+  "type": {
+    "disk": {
+      "capacity_enabled": true,
+      "free_space_less_than_percent": 15,
+      "io_above_percent": 85,
+      "io_enabled": true,
+      "io_over_period": "10m",
+      "rate_increase_percent": 20,
+      "rate_limit_mb_per_member": 5000000,
+      "rate_period_seconds": 600,
+      "rate_units": "mb"
+    },
+    "memory": {
+      "io_above_percent": 80,
+      "io_enabled": true,
+      "io_over_period": "10m",
+      "rate_increase_percent": 15,
+      "rate_limit_mb_per_member": 200000,
+      "rate_period_seconds": 600,
+      "rate_units": "mb"
+    }
+  }
+}
```

Comment:
This also did not work when I tried it:
```
 2024/06/28 10:27:55 Terraform plan | Error: Invalid value for input variable
 2024/06/28 10:27:55 Terraform plan | 
 2024/06/28 10:27:55 Terraform plan |   on schematics.tfvars line 27:
 2024/06/28 10:27:55 Terraform plan |   27: auto_scaling = {"type":{"disk":{"capacity_enabled":true,"free_space_less_than_percent":15,"io_above_percent":85,"io_enabled":true,"io_over_period":"10m","rate_increase_percent":20,"rate_limit_mb_per_member":5000000,"rate_period_seconds":600,"rate_units":"mb"},"memory":{"io_above_percent":80,"io_enabled":true,"io_over_period":"10m","rate_increase_percent":15,"rate_limit_mb_per_member":200000,"rate_period_seconds":600,"rate_units":"mb"}}}
 2024/06/28 10:27:55 Terraform plan | 
 2024/06/28 10:27:55 Terraform plan | The given value is not suitable for var.auto_scaling declared at
 2024/06/28 10:27:55 Terraform plan | variables.tf:160,1-24: attributes "disk" and "memory" are required.
```

Reply:
Oh!  I am looking into this.

Reply:
I am following this as per the [usage](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/database#sample-database-instance-by-using-auto_scaling) here. `mb` is a valid value here.